Makefile

```diff
@@ -64,6 +64,7 @@ test: generate lint manifests unit ## Run common developer tests
 .PHONY: unit
 unit: ## Run unit tests
 	go test ./... $(GO_TEST_FLAGS) -coverprofile $(COVER_PROFILE)
+	cd apis/ && go test ./... $(GO_TEST_FLAGS) -coverprofile $(COVER_PROFILE)
```

Comment:
what about unit-cover ? and is the fmt check needing the same trick ?

Reply:
As far as I can see, these are not running for apis/; 
`make unit-cover`
`make manifests`
`make test`
`make lint`

I could not run `make fmt-check` (probably I'm doing something wrong).